web/src/components/chat.tsx

```diff
@@ -0,0 +1,155 @@
+import { useState, useEffect } from "react";
```

Comment:
I moved the actual chat functionality/component bits to a separate component so it can be imported easier. This also will make it easier to move it over to the stampy-ui repo...